Sources/FirebaseFirestore/DocumentReference+Swift.swift

```diff
@@ -64,6 +68,56 @@ extension DocumentReference {
 
     return ListenerRegistration(boxed, instance)
   }
+
+  public func setData(_ data: [String: Any], merge: Bool = false, completion: ((NSError?) -> Void)?) {
+    let boxed = Unmanaged.passRetained(completion as AnyObject)
+    let converted = FirestoreDataConverter.firestoreValue(document: data)
+    let options = merge ? firebase.firestore.SetOptions.Merge() : firebase.firestore.SetOptions()
```

Comment:
Hmm, I take it that there is no public `OptionSet` that firebase exposes?

Reply:
Didn't seem like it unfortunately, also the api seems to just specify the single bool for 'merge' or not.

Reply:
Sadness 😞.